src/messages/create/publish.ts

```diff
@@ -57,6 +57,7 @@ export async function PutContentToStorageEngine<T>(configuration: PutConfigurati
         }
     }
     if (!configuration.inlineRequested) {
+        configuration.message.item_content = undefined;
```

Comment:
Is there a reason to explicitly set a property to `undefined`?

Reply:
In case someone set the message's item_content to some value, message validation on the CCN will fail for this case.